guidance:
  velocity_limits:
    vx: [-0.3, 0.3]
    vy: [-0.3, 0.3]
    vz: [-0.1, 0.1]
  pp:
    kappa: 0.1
  pid:
    x_axis:
      kp: 0.5
      ki: 0
      kd: 0
    y_axis:
      kp: 0.5
      ki: 0
      kd: 0
    z_axis:
      kp: 0.5
      ki: 0
      kd: 0

attitude_limits:
  pitch: [-5, 5]
  roll: [-5, 5]

reference_model:
  omegas: [10, 10]
  zetas: [1, 1]

pid:
  x_axis:
    kp: -7
    ki: -0.1
    kd: 0
  y_axis:
    kp: 7
    ki: 0.1
    kd: 0

linear_drag_model:
  drone_mass: 315
  g: 9.81
  dx: 0.08063504
  dy: 0.09929089

ipid:
  x_axis:
    alpha: 1
    kp: 7
    ki: 0.1
    kd: 0.5
  y_axis:
    alpha: 1
    kp: -7
    ki: -0.1
    kd: -0.5